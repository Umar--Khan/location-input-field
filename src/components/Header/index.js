const Header = () => {
  return (
    <header className='header-wrapper'>
      <img
        src='https://cdn.rcstatic.com/images/site_graphics/newsite/mobile/logos/rc-logo-small--white.svg'
        alt='rental cars logo'
      />
    </header>
  );
};

export default Header;
