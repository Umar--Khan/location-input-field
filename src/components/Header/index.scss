.header {
  background-color: $color-blue;
}

.header-wrapper {
  @include desktop {
    width: 60%;
    margin: 0 auto;
  }

  &__img {
    margin: $space-030;

    @include desktop {
      margin-left: 0;
    }
  }
}
