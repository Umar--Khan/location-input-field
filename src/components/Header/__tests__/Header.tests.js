import renderer from 'react-test-renderer';

import Header from 'components/Header';

describe('<Header />', () => {
  it('Render snapshot correctly', () => {
    const component = renderer.create(<Header />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
