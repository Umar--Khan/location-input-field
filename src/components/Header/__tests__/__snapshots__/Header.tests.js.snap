// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Render snapshot correctly 1`] = `
<header
  className="header"
>
  <div
    className="header-wrapper"
  >
    <img
      alt="rental cars logo"
      className="header-wrapper__img"
      src="https://cdn.rcstatic.com/images/site_graphics/newsite/mobile/logos/rc-logo-small--white.svg"
    />
  </div>
</header>
`;
